src/main/java/com/cloudbees/jenkins/GitHubPushTrigger.java

```diff
@@ -114,7 +118,12 @@ public void run() {
                         LOGGER.log(Level.WARNING, "Failed to parse the polling log",e);
                         cause = new GitHubPushCause(pushBy);
                     }
-                    if (job.scheduleBuild(cause)) {
+                    ParameterizedJobMixIn scheduler = new ParameterizedJobMixIn() {
```

Comment:
Because the object is created **only** to schedule a build, so it's used as a **scheduler**. Anyway, I can change it if you think is going to be clearer. Any suggestion?